shared/uavobjectdefinition/stabilizationsettings.xml

```diff
@@ -27,6 +27,9 @@
 		<field name="PitchPI" units="" type="float" elementnames="Kp,Ki,ILimit" defaultvalue="2.5,0,50" limits="%BE:0:20,%BE:0:20,"/>
 		<field name="YawPI" units="" type="float" elementnames="Kp,Ki,ILimit" defaultvalue="2.5,0,50" limits="%BE:0:20,%BE:0:20,"/>
 		<field name="GyroCutoff" units="Hz" type="float" elements="1" defaultvalue="55.0"/>
+		<field name="GyroButterworth" units="" type="enum" elements="1" options="FALSE,TRUE" defaultvalue="FALSE">
```

Comment:
I can turn this into an uint8 to select an order. Gonna take a while to brush my math up enough to construct filters beyond the second order (I know it's stacking biquads, but the coefficients).